site/en/docs/extensions/reference/action/index.md

```diff
@@ -2,9 +2,165 @@
 api: action
 ---
 
-!!!.aside.warning
-This documentation is in active development.
-We're planning to expand the introduction section to provide more context and
-guidance on how to use this API.  This work is tracked in issue
-[#193](https://github.com/GoogleChrome/developer.chrome.com/issues/193).
+You can use the `chrome.action` API to control the toolbar button for your
+extension in Chrome's UI.  The action icons are displayed in the browser
+toolbar, to the right of the omnibox (on LTR devices). After installation, by
```

Comment:
Does LTR mean left-to-right? If so, suggestion is "on left-to-right devices".

Reply:
Yep, Left-to-Right (as opposed to an RTL language, like Arabic, where much of the UI is reversed).  Updated.